uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/filters/due-expiry.filter.js

```diff
@@ -1,93 +1,90 @@
+/* eslint-env amd */
+
 define([
   'tasks-assignments/filters/filters'
 ], function (filters) {
   'use strict';
 
-  filters.filter('filterByDue',['$filter', '$log', function($filter, $log){
-    $log.debug('Filter: filterByDue');
-
-    return function(inputArr, type, dateRange) {
+  filters.filter('filterByDueExpiry', ['$filter', '$log', function ($filter, $log) {
+    $log.debug('Filter: filterByDueExpiry');
 
+    return function (inputArr, type, field, dateRange) {
       if (!inputArr) {
         return null;
       }
 
-      var filteredArr = [],
-        i = 0,
-        inputArrlen = inputArr.length,
-        date = new Date(),
-        today = date.setHours(0, 0, 0, 0);
+      var date = new Date();
+      var filteredArr = [];
+      var i = 0;
+      var inputArrlen = inputArr.length;
+      var today = date.setHours(0, 0, 0, 0);
+
+      field = field || 'activity_date_time';
```

Comment:
You are already sending the field name from the html, then why do we need to default a field name. Instead throw an error, if no field is specified.

Reply:
No need, removed. Thanks!